src/platforms/php/guides/laravel/performance/connect-services.mdx

```diff
@@ -4,17 +4,18 @@ sidebar_order: 40
 description: "Learn how to connect backend and frontend transactions."
 ---
 
-If you are also using Performance Monitoring for [JavaScript](/platforms/javascript/performance/) you can use a helper function to continue the trace started from your backend in order to [connect services](/platforms/javascript/performance/connect-services/#navigation-and-other-xhr-requests).
-Add the following line to your blade template rendering the `<head/>` of your page:
+If you are also using Performance Monitoring for [JavaScript](/platforms/javascript/performance/) you can use helper functions to continue the trace as well as the Dynamic Sampling Context, which contains additional trace-related data that is used for trace-based sampling, started from your backend in order to [connect services](/platforms/javascript/performance/connect-services/#navigation-and-other-xhr-requests).
+Add the following lines to your blade template rendering the `<head/>` of your page:
 
 ```php {filename:app.blade.php}
 <head>
 ...
 {!! \Sentry\Laravel\Integration::sentryTracingMeta() !!}
+{!! \Sentry\Laravel\Integration::sentryBaggageMeta() !!}
 ...
 </head>
 ```
 
-This helper function will render a meta tag similar to this `<meta name="sentry-trace" content="49879feb76c84743ba5034bd2d3f1ca3-7cb5535c930d4666-1"/>`, which our JavaScript SDK will pick up and continue the trace. Therefore, your frontend and your backend are connected by way of the same trace.
+These helper functions will render two meta tags similar to `<meta name="sentry-trace" content="49879feb76c84743ba5034bd2d3f1ca3-7cb5535c930d4666-1"/>` and <meta name="baggage" content="sentry-trace_id=49879feb76c84743ba5034bd2d3f1ca3,sentry-sample_rate=0.2,sentry-transaction=my-transaction,sentry-public_key=public,sentry-release=1.0.0,sentry-environment=prod>, which our JavaScript SDK will pick up and continue the trace and the Dynamic Sampling Context. Therefore, your frontend and your backend are connected by way of the same trace.
```

Comment:
The second half of this sentence seems to be missing some connecting words. This part specifically doesn't make sense: "which our JavaScript SDK will pick up and continue the trace and the Dynamic Sampling context"
Should it be "will pick up and then continue the trace and generate the Dynamic Sampling context." It's not clear what elements are taking which actions here.

```suggestion
These helper functions will render two meta tags similar to `<meta name="sentry-trace" content="49879feb76c84743ba5034bd2d3f1ca3-7cb5535c930d4666-1"/>` and `<meta name="baggage" content="sentry-trace_id=49879feb76c84743ba5034bd2d3f1ca3,sentry-sample_rate=0.2,sentry-transaction=my-transaction,sentry-public_key=public,sentry-release=1.0.0,sentry-environment=prod>`, which our JavaScript SDK will pick up and continue the trace and the Dynamic Sampling context. Therefore, your frontend and your backend are connected by way of the same trace.
```

Reply:
`sentry-trace` continues the trace, `sentry-baggage` will be used to propagate the Dynamic Sampling context.

Reply:
Okay what about this:
```suggestion
These helper functions will render two meta tags similar to `<meta name="sentry-trace" content="49879feb76c84743ba5034bd2d3f1ca3-7cb5535c930d4666-1"/>` and `<meta name="baggage" content="sentry-trace_id=49879feb76c84743ba5034bd2d3f1ca3,sentry-sample_rate=0.2,sentry-transaction=my-transaction,sentry-public_key=public,sentry-release=1.0.0,sentry-environment=prod>`. Our JavaScript SDK will pick these up and then continue the trace and propagate the Dynamic Sampling context. 
As a result, your frontend and your backend are connected by way of the same trace.
```